doc/modules/linear_model.rst

```diff
@@ -1000,6 +1000,8 @@ logistic regression, see also `log-linear model
   | `ElasticNet`   | :math:`\frac{1 - \rho}{2}\|W\|_F^2 + \rho \|W\|_{1,1}`                           |
   +----------------+----------------------------------------------------------------------------------+
 
+.. _Logistic_regression_solvers:
```

Comment:
Maybe drop the capital letter if possible.

Reply:
Done, I agree its weird with the capital. I initially thought I should copy what was used as reference header for logistic regression main section:

https://github.com/scikit-learn/scikit-learn/blob/8d01acfa72e03f07eb4db518c52a31d726585a0c/doc/modules/linear_model.rst?plain=1#L852